<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>morse</class>
 <widget class="QWidget" name="morse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>morse</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>664</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>请输入文字：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>速度（WPM）：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>音调（Hz）：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>600</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Version 1.1 By lijiaxuan 2024.08.11 https://www.yuanshen.dev/ https://github.com/lijiaxuan1811/Morse-Code-Tools</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
